platform/understand/externally-vs-platform-deployed.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Virtual Cluster Lifecycle Ownership
+sidebar_label: Virtual Cluster Lifecycle Ownership
+---
+
+# Virtual Cluster Lifecycle Ownership
+
+There are two main types of virtual clusters based on how they're managed: platform-managed and externally managed. These classifications determine how the clusters are created, updated, and integrated with the platform's features. Understanding the differences between these types is important for effective cluster management and utilization of platform capabilities.
```

Comment:
We wanted to specifically move away from managed to deployed to not confuse people on how you deploy would indicate how it's managed. 

Could you ask again in docs on which way we want to push forward on this concept? I did like the concept of "owned" vs. "managed" that you had pushed.

Reply:
We are settling on using `deployed` as the term.